### PR TITLE
change alert to matdialog in demo page

### PR DESCRIPTION
**Pull request by RizkyRajitha**
changed browser alert to matdialog in demo page

gfdemoalert

https://github.com/ghostfolio/ghostfolio/issues/1314

@dtslvr can you share you thoughts on this.

Thank you

### CODE REVIEW

File: apps/client/src/app/pages/demo/demo-page-dialog.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<h3 mat-dialog-title>{{text}}</h3>
```
Comment (dtslvr) [inline apps/client/src/app/pages/demo/demo-page-dialog.html:1]: Can you move this template to `libs/ui/src/lib/alert/alert.component.html`

File: apps/client/src/app/pages/demo/demo-page.component.ts [hunk 2]
```diff
@@ -17,28 +18,42 @@ export class DemoPageComponent implements OnDestroy {
...
-
-    this.router.navigate(['/']);
   }
 
   public ngOnDestroy() {
     this.unsubscribeSubject.next();
     this.unsubscribeSubject.complete();
   }
 }
+@Component({
```
Comment (dtslvr) [inline apps/client/src/app/pages/demo/demo-page.component.ts:43]: Can you move this component to `libs/ui/src/lib/alert/alert.component.ts`?

File: apps/client/src/app/pages/demo/demo-page-dialog.html (hunk 1 above)
Comment (dtslvr) [inline apps/client/src/app/pages/demo/demo-page-dialog.html:1]: Can you add an _Okay_ button?

```
<div mat-dialog-actions>
  <button cdkFocusInitial i18n mat-button mat-dialog-close>Okay</button>
</div>
```

**Review by dtslvr — CHANGES_REQUESTED** [review]

**Comment by dtslvr** [comment]
> @dtslvr can you share you thoughts on this.

Thanks for the update @RizkyRajitha 👍🏻  It is going in the right direction. I have added some comments.

**Comment by RizkyRajitha** [comment]
thank you @dtslvr for your feedback will update

**Comment by dtslvr** [comment]
Hey @RizkyRajitha

Are you planning to resume this pull request?

**Comment by RizkyRajitha** [comment]
hi @dtslvr ,
sorry for delayed reply , 
i am planning to work on these changes .

**Comment by dtslvr** [comment]
There has been no activity on this PR for an extended period. Closing it now.

**Pull request closed without merging**